gcloud/datastore/test___init__.py

```diff
@@ -151,27 +182,32 @@ def test_set_implicit_both_appengine_and_compute(self):
 
         APP_ENGINE_ID = 'GAE'
         APP_IDENTITY = _AppIdentity(APP_ENGINE_ID)
-        HTTPLIB = _Httplib('GCE')
+        connection = _HTTPConnection(200, APP_ENGINE_ID)
```

Comment:
This was intentionally `'GCE'` since the test is `...both_appengine_and_compute`